extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -294,31 +292,37 @@ public void getStreamCount() {
 
     public static class Invalid {
 
-        @BeforeClass
+        @BeforeAll
         public static void setUp() throws IOException {
             YoutubeParsingHelper.resetClientVersionAndKey();
             YoutubeParsingHelper.setNumberGenerator(new Random(1));
             NewPipe.init(new DownloaderFactory().getDownloader(RESOURCE_PATH + "invalid"));
             dummyCookie.put(YoutubeMixPlaylistExtractor.COOKIE_NAME, "whatever");
         }
 
-        @Ignore
-        @Test(expected = IllegalArgumentException.class)
+        @Disabled
+        @Test
         public void getPageEmptyUrl() throws Exception {
             extractor = (YoutubeMixPlaylistExtractor) YouTube
                     .getPlaylistExtractor("https://www.youtube.com/watch?v=" + VIDEO_ID
                             + "&list=RD" + VIDEO_ID);
-            extractor.fetchPage();
-            extractor.getPage(new Page(""));
+
+            assertThrows(IllegalArgumentException.class, () -> {
+                extractor.fetchPage();
+                extractor.getPage(new Page(""));
+            });
         }
 
-        @Test(expected = ExtractionException.class)
+        @Test
         public void invalidVideoId() throws Exception {
             extractor = (YoutubeMixPlaylistExtractor) YouTube
                     .getPlaylistExtractor("https://www.youtube.com/watch?v=" + "abcde"
                             + "&list=RD" + "abcde");
-            extractor.fetchPage();
-            extractor.getName();
+
+            assertThrows(ExtractionException.class, () -> {
+                extractor.fetchPage();
+                extractor.getName();
+            });
```

Comment:
I think `extractor.getName()` should be the only line included in the lambda, since it is the one that fails. If `fetchPage()` fails first, then `getName()` should be removed altogether